### ISSUE TITLE
Implement ElasticSearch for the search box

### ISSUE THREAD

**Issue by Polsaker** [Phuks-co/throat#25]

**Comment by dappercrow** [comment]
Would this just look like:
* Implement API calls and front-end mods necessary for communicating with search backend
* Update `config.yml` with examples on how an instance admin should configure their cluster to work with our implementation

?

**Comment by Polsaker** [comment]
Yeah plus all the changes to feed the data to elasticsearch as posts/comments are created, and maybe a script to do the initial dump to ES

**Comment by dappercrow** [comment]
OK, I haven't set this up before but it sounds like something interesting to learn.
